docs/explanation/remote.rst

```diff
@@ -92,6 +92,51 @@ End this test session preserving its data and launch a new one
   selection screens, in which case a similar test session is immediately
   started.
 
+
+Automatic session resume
+========================
+
+When the agent starts, it checks if there is a previous session that was not
+abandoned. If there is, and the session was a non-interactive one it resumes
+it. Otherwise, it waits for a Controller to connect and chose what to do.
+
+The outcome of the job that was last running before the session was
+interrupted is decided on the type of job it was running.
+
+The jobs marked with a ``noreturn`` flag are marked as passing, while other jobs
+are considered to have crashed (due to the interruption of the session like a
+reboot or a system stall).
+
+.. mermaid::
+
+  graph TD;
+
+  process("agent process starts")
+    load("load previous session")
+    resume("resume the previous session")
+    resume_crashed("resume the previous session")
+    interactive{"was the previous interactive"}
```

Comment:
las session interactive?
so it is the same as last job noreturn

Reply:
noreturn-ness and interactivity are orthogonal

(and I fixed the missing "session" word)

Reply:
I ment so the two lines are similar, not related. My objective is to make the diamonds also a little bit less HUGE because every char you add to them they seem to grow by 2m^2

Reply:
Understood. Trimmed the text to what you proposed.